rust-toolchain.toml

```diff
@@ -1,2 +1,2 @@
 [toolchain]
-channel = "nightly"
+channel = "channel = "nightly-2024-10-26"
```

Comment:
no more manual toolchain management, the project will use the right version of rust everytime ;) @0xkelvin